mne/_fiff/reference.py

```diff
@@ -155,6 +155,46 @@ def _apply_reference(inst, ref_from, ref_to=None, forward=None, ch_type="auto"):
     return inst, ref_data
 
 
+def _apply_dict_reference(inst, ref_dict):
+    """Apply a dict-based custom EEG referencing scheme."""
+    # Check that ref_dict channel exist in instance:
+    ref_dict_channels = list(
+        set(
+            list(ref_dict.keys())
+            + [item for sublist in ref_dict.values() for item in sublist]
```

Comment:
has this been tested when the dict value is a str not a list?

Reply:
Yeah good point, if you want to do bipolar, then you might pass a single channel. Will adjust